### PR TITLE
fix(opts): repair logging options for each call

### PR DESCRIPTION
**Pull request by smusali**
**fix(opts): repair logging options for each call**

Replacing back [this section](https://github.com/logdna/python/pull/70/files#diff-d98932f491dbf58581e648d2178063e919d9c1c80b58afa5e47bdc88c25211caL150-L184) which was removed in #70 

Semver: patch

---
**test(opts): test call-specific opts**

Semver: patch

### CODE REVIEW

File: logdna/logdna.py [hunk 1]
```diff
@@ -238,6 +238,15 @@ def emit(self, record):
...
         message['meta'] = sanitize_meta(message['meta'], self.index_meta)
+
+        opts = {}
+        if 'args' in record and not isinstance(record['args'], tuple):
+            opts = record['args']
+
+        for key in ['app', 'env', 'hostname', 'level', 'timestamp']:
+            if key in opts:
+                message[key] = opts[key]
+
```
Comment (teeohhem) [inline logdna/logdna.py:249]: Did these get removed before? Just curious what the root cause was.

File: logdna/logdna.py [hunk 2]
```diff
@@ -238,6 +238,15 @@ def emit(self, record):
...
         self.buffer_log(message)
```
Comment (teeohhem) [inline logdna/logdna.py:250]: Can we get a test around this please?

File: logdna/logdna.py (hunk 1 above)
Comment (smusali) [reply logdna/logdna.py:249]: @teeohhem, while working on specifically meta fields, this section was removed and gone unnoticed

File: logdna/logdna.py (hunk 2 above)
Comment (smusali) [reply logdna/logdna.py:250]: Done!

**Review by teeohhem — APPROVED** [review]

**Pull request merged**